src/commands.rs

```diff
@@ -2877,6 +2880,26 @@ fn cmd_resolve(
                 }),
         ));
     }
+    if args.list {
+        let mut formatter = ui.stdout_formatter();
+        let formatter = formatter.as_mut();
+        for (repo_path, _conflict_id) in conflicts {
+            // TODO: Similar to `jj diff --short`, insert a few letters
+            // before the filename to indicate the kind of conflict.
+            // E.g. we could have a letter per add : `FF` is a usual conflict
+            // between two versions of a file, `FD` is a file vs directory,
+            // `FFF` for a merge of three conflicting versions. Additionally,
+            // if (# removes) + 1 > (# adds), this indicates the file was deleted
+            // in some versions of the conflict. Perhaps that should be `R` for removed.
```

Comment:
I attempted something like that for the `jj status` output in #820, but it's probably too hard to read for regular users. See the discussion there. Maybe it's better to describe the conflict in a more verbose but clear way.

Reply:
Yeah, I thought this summary message is a more appropriate place for this (since people are asking for details about conflicts), but I'm not entirely sure. In any case, I agree there should be a more verbose one (though I'm not sure if/when I'll get to it -- I need some inspiration for what it should say).

After your comment above, I think `--list` should be the more verbose one, and `--summary` should match `diff --summary`. For now, I'll keep it as is.